prescient/engine/egret/data_extractors.py

```diff
@@ -81,119 +99,209 @@ def is_generator_on(self, sced: OperationsModel, g: G) -> bool:
         return sced.data['elements']['generator'][g]['initial_status'] > 0
 
-    def get_fixed_costs(self, sced: OperationsModel) -> float:
+    @staticmethod
+    def get_fixed_costs(sced: OperationsModel) -> float:
         total = 0.
         for g,g_dict in sced.elements(element_type='generator', generator_type='thermal'):
             total += g_dict['commitment_cost']['values'][0]
         return total
 
-    def get_variable_costs(self, sced: OperationsModel) -> float:
+    @staticmethod
+    def get_variable_costs(sced: OperationsModel) -> float:
         total = 0.
         for g,g_dict in sced.elements(element_type='generator', generator_type='thermal'):
             total += g_dict['production_cost']['values'][0]
         return total
 
-    def get_power_generated(self, sced: OperationsModel, g: G) -> float:
+    @staticmethod
+    def get_power_generated(sced: OperationsModel, g: G) -> float:
         return sced.data['elements']['generator'][g]['pg']['values'][0]
 
-    def get_power_generated_T0(self, sced: OperationsModel, g: G) -> float:
+    @staticmethod
+    def get_power_generated_T0(sced: OperationsModel, g: G) -> float:
         return sced.data['elements']['generator'][g]['initial_p_output']
 
-    def get_load_mismatch(self, sced: OperationsModel, b: B) -> float:
+    @staticmethod
+    def get_load_mismatch(sced: OperationsModel, b: B) -> float:
         return sced.data['elements']['bus'][b]['p_balance_violation']['values'][0]
 
-    def get_positive_load_mismatch(self, sced: OperationsModel, b: B):
-        val = self.get_load_mismatch(sced, b)
+    @staticmethod
+    def get_positive_load_mismatch(sced: OperationsModel, b: B):
+        val = ScedDataExtractor.get_load_mismatch(sced, b)
         if val > 0:
             return val
         return 0
 
-    def get_negative_load_mismatch(self, sced: OperationsModel, b: B):
-        val = self.get_load_mismatch(sced, b)
+    @staticmethod
+    def get_negative_load_mismatch(sced: OperationsModel, b: B):
+        val = ScedDataExtractor.get_load_mismatch(sced, b)
         if val < 0:
             return -val
         return 0
 
-    def get_max_power_output(self, sced: OperationsModel, g: G) -> float:
+    @staticmethod
+    def get_max_power_output(sced: OperationsModel, g: G) -> float:
         p_max = sced.data['elements']['generator'][g]['p_max']
         if isinstance(p_max, dict):
             p_max = p_max['values'][0]
         return p_max
 
-    def get_max_power_available(self, sced: OperationsModel, g: G) -> float:
+    @staticmethod
+    def get_max_power_available(sced: OperationsModel, g: G) -> float:
         gdata = sced.data['elements']['generator'][g]
         val = gdata['headroom']['values'][0] \
-                + self.get_power_generated(sced, g)*int(self.is_generator_on(sced,g))
+                + ScedDataExtractor.get_power_generated(sced, g)*int(ScedDataExtractor.is_generator_on(sced,g))
         if 'rg' in gdata:
```

Comment:
Should this be changed from https://github.com/grid-parity-exchange/Egret/pull/275?